### PR TITLE
Add SceneLab

### PR DESCRIPTION
**Pull request by nilsmehlhorn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -848,6 +848,7 @@ Table of Contents
   * [Zeplin](https://zeplin.io/) — Designer and developer collaboration platform. Show designs, assets and styleguides. Free for 1 project.
   * [Pixelixe](https://pixelixe.com/) — Create and edit engaging and unique graphics and images online.
   * [Responsively App](https://responsively.app) - A free dev-tool for faster and precise responsive web application development.
+  * [SceneLab](https://scenelab.io) - Online mockup graphics editor with an ever-expanding collection of free design templates
```
Comment (ripienaar) [inline README.md:851]: would you mind mentioning the limitations? like 800px size export and watermarks is quite a big deal - almost to the point where I would say its just not suitable for the list...its not exactly functional while free

Comment (nilsmehlhorn) [reply README.md:851]: The resolution limitations do not apply for free templates, you can export these in all available resolutions without any watermarks

Comment (ripienaar) [reply README.md:851]: ah ok, cool

Comment (nilsmehlhorn) [reply README.md:851]: sorry, I messed up, the resolution limits still apply atm (but we're planning to open this up soon). Free templates don't contain watermarks though.

**Pull request merged**